dangerfile.ts

```diff
@@ -71,7 +71,7 @@ if (componentSourceFilesModified && !snapshotsModified) {
 const componentFilesModified = fileChanges.some(
   (filePath) =>
     // packages/(one or more chars)/src/(one or more chars).(js or ts or tsx)
-    filePath.match(/packages\/.*bpk-component.+\/src\/.+\.(js|ts|tsx)$/) &&
```

Comment:
How come we changed this to not include JS whilst we still have JS files in Backpack?

Reply:
simply because the JS files won't have type files so then there's nothing to update